packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -0,0 +1,272 @@
+    actionHandlers: {},
+    rtl: context.rtl,
+  });
+
+  const { classes } = useStyles<DatepickerStylesProps>(Datepicker.displayName, {
+    className: datepickerClassName,
+    mapPropsToInlineStyles: () => ({
+      className,
+      design,
+      styles,
+      variables,
+    }),
+    rtl: context.rtl,
+  });
+
+  const handleChange: DatepickerCalendarProps['onDaySelect'] = (e, data) => {
+    const targetDay = data.value;
+    setSelectedDate(targetDay.originalDate);
+    setOpen(false);
+
+    _.invoke(props, 'onDayChange', e, { ...props, value: targetDay });
+  };
+
+  const element = (
+    <Ref innerRef={datepickerRef}>
+      {getA11yProps.unstable_wrapWithFocusZone(
+        <ElementType
+          {...getA11yProps('root', {
+            className: classes.root,
+            ...unhandledProps,
+          })}
+        >
+          <Input readOnly onClick={showCalendarGrid} value={valueFormatter(selectedDate)} />
+          <Popup
+            open={open}
+            onOpenChange={(e, { open }) => setOpen(open)}
+            content={
+              <DatepickerCalendar {...calendarOptions} onDaySelect={handleChange} localizedStrings={DEFAULT_STRINGS} />
+            }
+            trapFocus
+          >
+            <Button icon={<CalendarIcon />} title="Open calendar" iconOnly />
+          </Popup>
+        </ElementType>,
+      )}
+    </Ref>
+  );
+  setEnd();
+  return element;
+};
+
+Datepicker.displayName = 'Datepicker';
+
+Datepicker.propTypes = {
+  ...commonPropTypes.createCommon(),
+
+  minDate: PropTypes.instanceOf(Date),
+  maxDate: PropTypes.instanceOf(Date),
+  restrictedDates: PropTypes.arrayOf(PropTypes.instanceOf(Date)),
+
+  firstDayOfWeek: PropTypes.oneOf(Object.keys(DayOfWeek).map(name => DayOfWeek[name])),
+  firstWeekOfYear: PropTypes.oneOf(Object.keys(FirstWeekOfYear).map(name => FirstWeekOfYear[name])),
+  dateRangeType: PropTypes.oneOf(Object.keys(DateRangeType).map(name => DateRangeType[name])),
+  daysToSelectInDayView: PropTypes.number,
+  today: PropTypes.instanceOf(Date),
+  showWeekNumbers: PropTypes.bool,
+  workWeekDays: PropTypes.arrayOf(PropTypes.oneOf(Object.keys(DayOfWeek).map(name => DayOfWeek[name]))),
+
+  localizedStrings: PropTypes.object as PropTypes.Validator<IDateGridStrings>,
+
+  format: PropTypes.func,
+  parse: PropTypes.func,
+
+  disabled: PropTypes.bool,
+  isRequired: PropTypes.bool,
+  onDayChange: PropTypes.func,
+  goToToday: PropTypes.string,
+  placeholder: PropTypes.string,
+  renderCell: PropTypes.func,
+  renderHeaderCell: PropTypes.func,
+};
+
+Datepicker.defaultProps = {
+  accessibility: datepickerBehavior,
+  firstDayOfWeek: DayOfWeek.Monday,
+  firstWeekOfYear: FirstWeekOfYear.FirstDay,
+  dateRangeType: DateRangeType.Day,
+};
+
+Datepicker.handledProps = Object.keys(Datepicker.propTypes) as any;
+
+Datepicker.create = createShorthandFactory({ Component: Datepicker });
+
+/**
+ * A Datepicker is used to display dates.
+ */
+export default Datepicker;
```

Comment:
@rymeskar please could you sync with @layershifter regarding this, since he has an open PR #13934 to remove all `export default` instances
I think it's safe to do that for this component right away